ionex2kml/README.md

```diff
@@ -0,0 +1,51 @@
+ionex2kml
+=========
+
+`ionex2kml` is a small application to convert a IONEX file into KML format.
+
+It is a convenient method to visualize TEC maps using third party tools.
+
+Getting started
+===============
+
+Provide a IONEX file with `-i`:
+
+```bash
+    ionex2kml -i /tmp/ionex.txt
+```
+
+Input IONEX files do not have to follow naming conventions.
+
+This tool will preserve the input file name by default, just changing the internal
+format and file extension. In the previous example we would get `/tmp/ionex.kml`.
+
+Define the output file yourself with `-o`:
+
+```bash
+    ionex2kml -i jjim12.12i -o /tmp/test.kml
+```
+
+Each Epoch is put in a dedicated KML folder.  
+
+Equipotential TEC values
+========================
+
+When converting to KML, we round the TEC values and shrink it to N equipotential areas.  
+In other words, the granularity on the TEC visualization you get is max|tec|/N where max|tec|

```

Comment:
https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions might make this easier to read.

Reply:
Do you know if latex like equations can be written in docrs too ? At the time I looked at it, there was a tweak involving storing html pages if I remember correctly, seemd too complicated

Reply:
No, not really: https://crates.io/crates/rust-latex-doc-minimal-example, https://docs.rs/katex-doc/, https://docs.rs/pwnies/0.0.13/pwnies/.